front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/BannerImageFields/ImageUploader.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
Extracted from BannerImageFields. Still not ideal abstraction, just slightly better.